test_runner/fixtures/benchmark_fixture.py

```diff
@@ -485,7 +489,6 @@ def pytest_terminal_summary(
 
     out_dir = config.getoption("out_dir")
     if out_dir is None:
-        warnings.warn("no out dir provided to store performance test results")
```

Comment:
I've often had this dirty/uncommited locally, this is very welcome indeed, never got to asking if we really need this.